sass/layout/_markdown.scss

```diff
@@ -204,6 +217,7 @@ h6.markdown__heading {
         padding: 6.5px;
```

Comment:
change to `padding: 8.5px`

This will prevent the hover actions from spanning outside the bounds of single line code blocks